universal-login-example/src/App.tsx

```diff
@@ -2,22 +2,27 @@ import React from 'react';
 import Web3 from 'web3';
 import {ULWeb3Provider} from '@universal-login/web3';
 
-const universalLogin = new ULWeb3Provider(new Web3.providers.HttpProvider('https://rinkeby.infura.io'));
-const web3 = new Web3(universalLogin);
+const config = {
+  relayerUrl: process.env.RELAYER_URL!,
+  jsonRpcURl: process.env.JSON_RPC_URL!,
+  ensDomains: [process.env.ENS_DOMAIN_1!],
+};
 
-web3.eth.getAccounts()
-  .then(accounts => console.log('accounts', accounts))
-  .catch(console.error);
+const universalLogin = new ULWeb3Provider(
```

Comment:
ULWeb3Provider should have one parameter ‘config’.
Default value should be mainnet config. There should be one for each: mainnet, local dev and each testnet.
There should be two parameters ‘jsonRpcUrl’ and ‘web3provider’.
User can one them to specify how to connect to node.